flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/config/FlinkConfigBuilder.java

```diff
@@ -243,7 +243,8 @@ protected FlinkConfigBuilder applyPodTemplate() throws IOException {
 
     protected FlinkConfigBuilder applyIngressDomain() {
         // Web UI
-        if (spec.getIngress() != null) {
+        if (spec.getIngress() != null
```

Comment:
I think we can get rid of `applyIngressDomain()` altogether, this default configuration is already applied in method `applyFlinkConfiguration()`, where it's only set if the configuration is not yet set by the user.

Reply:
OK, I have removed `applyIngressDomain()`